server/createBook.js

```diff
@@ -0,0 +1,28 @@
+import {query} from './query.js';
+
+//Book creator endpoint
+
+export const createBook = async (req, res) => {
+  if (!req.body)
+    return res.status(406).send('What are you even trying to do here?');
+
+  const {title, createdByUser} = req.body;
```

Comment:
`createdByUser` should come from `req.user.email`, not from the request body. Never trust data that came from the client.